CHANGELOG.md

```
@@ -1,5 +1,13 @@
 # Changelog of the Pytest Coverage Comment
 
+## [Pytest Coverage Comment 1.12.1](https://github.com/MishaKav/pytest-coverage-comment/tree/v1.12.1)
+
+**Release Date:** 2026-08-30
+
+#### Changes
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)